Main Server/components/layout.js

```diff
@@ -67,8 +68,18 @@ export default function Layout({ children, home, searchEnabled, server, relative
         <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet" />
         <title>Dose</title>
       </Head>
-      <header style={relative !== undefined ? {position: 'relative'} : {}}>
-        <img className={styles.logo} src={`${process.env.NEXT_PUBLIC_SERVER_URL}/images/logo.png`} layout="fill"></img>
+      <header style={relative !== undefined ? {position: 'relative'} : {alignItems: 'center', display: 'flex'}}>
+        {!shouldShowMenu &&

```

Comment:
Can menu be it's own component?
so like:
`<dose-menu display=${shouldShowMenu}></dose-menu>`

Reply:
no

Reply:
:(